beacon/endpoints/rest/schemas/alternative.py

```diff
@@ -82,24 +82,23 @@ def build_phenopackets_meta_data_block(ontologies_used):
         'created': now.strftime(conf.datetime_format),  # required
         'created_by': conf.beacon_name,  # required
         # 'submitted_by': None,
-        'resources': ontologies_used,  # required
+        'resources': jsonb(ontologies_used),  # required
         # 'updates': None,
         # 'phenopacket_schema_version': None,
         # 'external_references': None,
     }
 
 
-def get_sampled_tissue(hstore, schema_name):
+def get_sampled_tissue(sample_origins): #hstore, schema_name):
     """
     Returns the first element of the list.
     This is because this field might have many values but phenopackets only accepts one.
     """
-    sample_origins = list(filter_hstore(hstore, schema_name) or [])
-    return sample_origins[0] if sample_origins else None
+    json_array = json.loads(sample_origins)
```

Comment:
I don't understand why you parse the json variable to use the first element if you had one. Why don't you arrange so that the response is already correct. After all, the DB did format it as json, so why not updating there, in the db, so you get the value here.

We had removed the json parsing entirely and now you are putting it back

Reply:
You might have many sample origins and, currently, _just in the case of using the phenopacket schema_, only one sample origin must be returned. I don't think I can do this in the Postgres function, but I'll check.

Reply:
Solved in new commit.